apps/nextjs/src/app/swap/SwapTokens.tsx

```diff
@@ -61,6 +65,34 @@ export const SwapTokens = () => {
   });
   const isDebouncing = useDebounce(sellAmount, 350) !== sellAmount;
 
+  useEffect(() => {
```

Comment:
![image](https://github.com/BibliothecaDAO/RealmsWorld/assets/90423049/6b8702fc-5a0a-46ce-a6f5-53db8fb2ccfa)
At the moment the end swap total received is incorrect (e.g Beast costing 500, I have 164 exact balance and the amount received is 335). From a brief look I think this is from the logic of:

1. fetching the quotes for a sale of the lords qty
2. setting the sellAmount to the result of step 1

This is probably missing the impact of Avnu fees (0.02% is charged for direct routes, and 0.15% for complex routing). You may be able to calculate the fees exactly from the avnu fetchquotes response - but may be easier to just add a small % buffer.

Also you can see in the photo above the switch direction Icon is misplaced - given this is a one way purchase only - it is probably best to add a prop to remove the button completely (i.e cant switch back to selling Lords).